### PR TITLE
Updates to markdown linting

### PR DESCRIPTION
**Pull request by maxwondercorn**
Multiple updates to the markdown linting setup

- Moved selected words from the local dictionary to ember-dictionary
- Updated .remarkignore to lint only 3.8.x onward (2 LTS releases)
- Bumped ember-dictionary version to latest
- Bumped retext-repeated-words version to latest

### CODE REVIEW

File: .local.dic [hunk 1]
```diff
@@ -136,6 +124,4 @@ unsilence
 untrusted
 usecase
 viewport
-VoiceOver
-WAI-ARIA
-WCAG
+VoiceOver
```
Comment (chancancode) [inline .local.dic]: Can we have a trailing newline here?

File: .remarkignore [hunk 2]
```diff
@@ -27,4 +27,8 @@ guides/v2.18.0/**/*.md
 guides/v3.0.0/**/*.md
 guides/v3.1.0/**/*.md
 guides/v3.2.0/**/*.md
-guides/v3.3.0/**/*.md
\ No newline at end of file
+guides/v3.3.0/**/*.md
+guides/v3.4.0/**/*.md
+guides/v3.5.0/**/*.md
+guides/v3.6.0/**/*.md
+guides/v3.7.0/**/*.md
```
Comment (chancancode) [inline .remarkignore]: Same re: newline

**Comment by chancancode** [comment]
@maxwondercorn do you think you can help with https://github.com/ember-learn/super-rentals-tutorial/issues/13?

That workflow is that we write special "runnable" markdowns in that repo. When we merge a pull request there, CI will build that into "regular" markdown, and integrate the result into this repo, e.g. 
https://github.com/ember-learn/guides-source/pull/1126. Because we don't have linting setup in super-rentals-tutorial, often we will merge a PR over there, only to find out when it's being integrated that it failed some lints (repeated words and what not), and had to fix that in a second PR on super-rentals-tutorial.

It would be nice to "roughly mirror" (i.e. duplicate) the linting setup we used here into super-rentals-tutorial, so the CI there will catch most of the mistakes before we merge it there.

I'm brining it up since you seem to know how these things are setup here 😄

**Comment by chancancode** [comment]
@jenweber should this target the octane branch?

**Comment by jenweber** [comment]
@chancancode it's ok for this to target `master`. We didn't freeze PRs to 3.13 yet.

**Comment by maxwondercorn** [comment]
@chancancode I'll take a look at it.

File: .local.dic (hunk 1 above)
Comment (maxwondercorn) [reply .local.dic]: Trailing newline added

File: .remarkignore (hunk 2 above)
Comment (maxwondercorn) [reply .remarkignore]: Trailing newline added

**Comment by maxwondercorn** [comment]
Looks like travis errors were timeouts

**Comment by chancancode** [comment]
@maxwondercorn Thank you for working on this! Do you mind rebasing against master?

**Comment by maxwondercorn** [comment]
@chancancode I will get the rebase this afternoon

**Comment by maxwondercorn** [comment]
Added words from super-rentals-tutorial and rebased against master.  This PR removed some words from the local dictionary and put them into the main ember-dictionary.  

If someone updates their local guides-source copy with a `git pull` they should run `npm install` to pull down the latest dictionary.

**Comment by chancancode** [comment]
@maxwondercorn Thank you for working on this (and https://github.com/ember-learn/super-rentals-tutorial/issues/13). Sorry it had taken so long to merge this. The builds are pretty broken right now 😐

**Comment by chancancode** [comment]
@maxwondercorn I think the next step is to merge these changes into the octane branch (which is where the new tutorial is hosted). Do you think you have cycles to work on that next?

**Comment by maxwondercorn** [comment]
@chancancode Updating the octane branch should be pretty straightforward. Diff the local dictionaries and update the linting dependencies should be all that's required.  I should get to it by the end of the week

**Pull request merged**